patroni/postgresql.py

```diff
@@ -292,6 +293,11 @@ def reload_config(self, config):
     def pending_restart(self):
         return self._pending_restart
 
+    @staticmethod
+    def configuration_allows_rewind(data):
+        return data.get('Current wal_log_hints setting', 'off') == 'on' \
+            or data.get('Data page checksum version', '0') != '0'
```

Comment:
> we didn't check full_page_writes, but we should.

How exactly do you want to check it? Current setting on master? And what if it was enabled just a few moments ago?